transform/mattermost-analytics/models/staging/stripe/_stripe__models.yml

```diff
@@ -71,105 +71,151 @@ models:
 
   - name: stg_stripe__subscriptions
-    description: Staging table for Stripe subscriptions
+    description: |
+      Staging table for Stripe subscriptions. Note that some of these subscriptions might have been created for one of
+      these reasons:
+      - Admin generated license
+      - SFDC migrated licenses
+      - Custom backfilling
+      
+      More details are available in Confluence:
+      - [Self-hosted](https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2453733377/Self-Hosted+Self+Serve+Subscription+Renewals)
+      - [Cloud](https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2455633921/Cloud+Mattermost+Cloud+Subscriptions+with+history)
+
     columns:
-      - name: billing
-        description: The billing type of the subscription (e.g., 'charge_automatically')
-      - name: billing_cycle_anchor
-        description: The date at which the current billing period ends
-      - name: cancel_at_period_end
-        description: A boolean indicating whether the subscription will be canceled at the end of the current billing period
-      - name: collection_method
-        description: The payment collection method for the subscription (e.g., 'charge_automatically')
-      - name: created_at
-        tests:
-          - not_null
-        description: The timestamp when the subscription was created
-      - name: current_period_end
-        description: The end date of the current billing period
-      - name: current_period_start
-        description: The start date of the current billing period
-      - name: customer_id
-        description: The customer to whom the subscription belongs
       - name: subscription_id
+        description: The ID of the subscription.
         tests:
           - not_null
           - unique
-        description: The ID of the subscription
-      - name: invoice_customer_balance_settings
-        description: The settings for automatic customer balance management
-      - name: items
-        description: An array of subscription line items
+      - name: customer_id
+        description: The customer to whom the subscription belongs.
       - name: latest_invoice_id
-        description: The ID of the most recent invoice for the subscription
-      - name: livemode
-        description: A boolean indicating whether the subscription is in live mode
-      - name: cws_dns
-        description: The DNS of the subscription
-      - name: cws_installation
-        description: The installation ID of the subscription
-      - name: cws_installation_state
-        description: The installation state of the subscription
-      - name: billing_type
-        description: The type of billing for the subscription
+        description: The ID of the most recent invoice for the subscription.
       - name: current_product_id
-        description: The ID of the current product for the subscription
+        description: The ID of the current product for the subscription.
```

Comment:
1/5 - if we don't already, might make sense to document the difference between "current product ID" and "product ID" below.

Reply:
This seems to be a remnant of monthly cloud subscriptions. Given it's not affecting the current PR, let's discuss how to handle it in a separate PR or when we start tackling cloud installations.